parsec/core/backend_connection/authenticated.py

```diff
@@ -199,7 +219,7 @@ def __init__(
         self._status_event_sent = False
         self._cmds = BackendAuthenticatedCmds(addr, self._acquire_transport)
         self._manager_connect_cancel_scope = None
-        self._monitors_cbs: List[Callable[..., None]] = []
+        self._monitors_cbs: List[Callable[..., Any]] = []
```

Comment:
All monitors callbacks do not return the same type? That sounds fishy..

Also, maybe another good use case for `Protocol`